docs/modules/5_path_planning/path_planning_main.rst

```diff
@@ -31,4 +31,5 @@ Path planning is the ability of a robot to search feasible and efficient path to
    hybridastar/hybridastar
    frenet_frame_path/frenet_frame_path
    coverage_path/coverage_path
-   elastic_bands/elastic_bands
\ No newline at end of file
+   elastic_bands/elastic_bands
+   time_based_grid_search/time_based_grid_search
```

Comment:
Sorry, could you please move this document under the
`grid_base_search/grid_base_search`
not the bottom?

Reply:
Sure thing! Moved:
https://github.com/AtsushiSakai/PythonRobotics/pull/1170/commits/f2ac5d9868878edf6c87b087b117e75ded88aad1#diff-9c0354fe10d203f73da8f1b61aaee6302cf944add0f7286782ca641f20dbab26R15